build/restore.ps1

```diff
@@ -1,14 +1,17 @@
 $root = $PSScriptRoot
-$packages = "$root\..\packages"
-
 $nuget = & "$root\Get-Nuget.ps1"
 
-New-Item $packages -ItemType Directory -ErrorAction Ignore
-
 # Restoring NuGet packages into this specific directory because the automated
-# build definition needs to be able to find the Visual Studio Test Adapter 
+# build definition in VSTS needs to be able to find the Visual Studio Test Adapter 
 # (xunit.runner.visualstudio) in a known location
-& $nuget restore $root\..\PortabilityTools.sln -PackagesDirectory $packages
+if($env:NuGetPackagesDirectory)
```

Comment:
You'll need to update the build def to set it, but it seems to be used deep in the msbuild scripts to determine where to resolve packages.